### DISCUSSION TITLE
Tab sort by descending weight

### DISCUSSION THREAD

**Discussion by FuckingDeveloper** [Zrips/CMI#5683] [Enhancement]
Hi, are you going to update the CMI plugin, while adding sorting in the tab by the same weight from Luckperms? This would be convenient, because you are positioning yourself as a plugin that replaces many other plugins in the aggregate.

It just doesn't look good right now. That is, for example, the first has a weight of 100, the second weight of 0, the third weight of 40, the fourth weight of 0. I would like to sort them in a convenient order for the eye.

I think that this is easy to implement on the part of the plugin, and in general it is implemented in 15-30 minutes.

![Tab-list](https://user-images.githubusercontent.com/26286168/120166944-d3dc7e80-c205-11eb-992c-286a7a0a26ac.png)

**Comment by CloudeLecaw** [comment]
May easy to add but was asked now more then 10 times to be added and yes also in the last two years.
So i dont think it will be added now.
Further i think this will be just moved to the Suggestion site like the other then that asked for playernamesorting in tab.

**Comment by mrfloris** [comment]
Issues is now used for Bug reports, Discussions is used for feature requests;
https://github.com/Zrips/CMI/discussions
